front/src/components/nft-card-information/NftCardInformation.tsx

```diff
@@ -10,23 +11,54 @@ const NftCardInformation = ({ data }: NftCardInformationProps) => {
   const { classes } = useStyles();
 
   return (
-    <div>
-      <GCText weight={500}>{data.name}</GCText>
-      <GCText weight={400} className={classes.priceText}>
-        Price: {data.mintingPrice}
+    <>
+      <GCText weight={700} size={32} color={"black"}>
+        {data.name}
       </GCText>
-      <div className={classes.listSize}>
-        <GCList size={12}>
-          <GCListItem>Rarity: {data.rarity}</GCListItem>
-          <GCListItem>Collection: {data.collection}</GCListItem>
-          <GCListItem>First owner: {data.firstOwner}</GCListItem>
-          <GCListItem>Creation date: {data.mintingDate}</GCListItem>
-        </GCList>
+      <GCText weight={400} size={16} color={"black"} className={classes.collection}>
+        Collection:
+        <GCText size={16} weight={700} color={"darkviolet"}>
```

Comment:
GCText is not adaptive with this direct setting of weight and size. Upgrade GCText component to correct library version in this taskhttps://app.clickup.com/t/376353u , and use it here